src/ui/sass/modules/_Pagination.scss

```diff
@@ -55,10 +55,28 @@ $pagination-color-hover: var(--yxt-color-text-secondary) !default;
     }
   }
 
-  #active-page {
+  &--active-all {
     background-color: var(--yxt-pagination-color-active-page);
   }
 
+  &--active-mobile {
```

Comment:
I think we want this all to be one word camelCase to fit the BEM style e.g. `--activeMobile`